crates/serpen/tests/fixtures/ast_rewriting/alias_transformation_test/utils/config_manager.py

```
@@ -0,0 +1,14 @@
+"""
+Configuration management utilities.
+Used to test from-import alias transformations.
+"""
+
+
+def load_config(config_file):
+    """Load configuration from a file (simulated)."""
+    return {"file": config_file, "loaded": True, "settings": {"debug": False}}
+
+
+def save_config(config_data, config_file):
```

Ruff (0.11.9)

Missing return type annotation for public function `save_config`

Add return type annotation: `str`

(ANN201)